guides-and-standards/standard-public-repos.md

```diff
@@ -0,0 +1,45 @@
+
+## Standards for organization and documentation
+
+- **Repository organization**: The repository should be well-organized with descriptive file names and a clear directory structure.
+  - Scripts, notebooks, and data files should generally be in separate directories.
+  - Directory names and filenames should not include spaces or special characters.
+  - Avoid very long scripts or notebooks. If a script or notebook is more than 500-1000 lines long, consider breaking it up into smaller scripts or functions.
+- **Descriptive README**: The repository should include a thorough and clear README that explains the context and purpose of the repo, what the code does, and how to use it.
+  - The README should explain how to create the same environment that was originally used to run the code. This includes the operating system you ran the code on, the software you installed, and the version of the software.
+  - The README should include an explanation of how to access or prepare any data needed to run the code.
+- **Dependency versions**: The exact versions of all direct dependencies (i.e., Python and/or R packages) used in the repository should be listed. Typically, this takes the form of a Conda environment file (usually `envs/dev.yml`).
+  - For Python projects, dependencies can also be listed in a `requirements.txt` file or in `pyproject.toml` if you are using a package manager like `poetry`.
+  - Caveat: for libraries or packages, it may be more appropriate to specify a minimum version, rather than an exact version, for some dependencies.
+- **No large files**: The repository should not include files larger than 100MB, even if they are necessary for the code to run. If you need to include large files, they should be hosted on Zenodo or an appropriate domain-specific archive (e.g., the SRA) and a link to the files should be included in the README.
+- **No sensitive information**: The repository should not include any sensitive or private information like API keys or access credentials. If the code requires sensitive information to run, it should be stored in a separate configuration file that is not included in the repository. If you accidentally added sensitive information to your repository, please reach out to the software team for help. **It is not sufficient to delete the sensitive information in a subsequent commit**, as the information will still exist in the commit history.
+- **License**: The repository should use the MIT license.
+  - Caveat: make sure that this license is compatible with the licenses of the software/dependencies used in your project (it usually is). Please check with the software team if you are unsure about this.
+
```

Comment:
consider adding a bullet about required repo settings: branch protection for `main`, requiring PRs, multiple owners/admins.

Reply:
+1 on this. Using the defaults here should be okay: https://www.notion.so/arcadiascience/How-to-manage-paired-public-and-private-GitHub-repos-2c3e96d6097943a099c3cdc0ab8789df?pvs=4

Reply:
For now I linked to that notion document, but I think ideally the branch protection rules should be moved out of that notion page (where they're kind of buried and also tangential to the main document) and into the source-control standard document (since they apply to all repos, not just public repos).

Reply:
They're also in the source-control standard document

Reply:
Ah right. It looks like they aren't quite the same as what's on the notion page, so it would be good to de-duplicate them. I'll make a note to come back to this.